query-string/query-string.d.ts

```diff
@@ -9,14 +9,14 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): any;
+    export function parse(str: string): {[key: string]: string};
 
     /**
      * Stringify an object into a query string, sorting the keys.
      *
      * @param obj
      */
-    export function stringify(obj: any, options?: {strict: boolean}): string;
+    export function stringify(obj: {[key: string]: string|number|boolean}, options?: { strict: boolean }): string;
```

Comment:
Or to be more specific, because `any | any[]` doesn't really do much. If a type is `any`, it will match an array as well.

```ts
type value = string | number | boolean;

export function stringify(obj: {[key: string]: value | value[]}, options?: {strict: boolean}): string
```

I would suggest adding more tests as well to test all these use cases.